src/desktop/models/fair.coffee

```diff
@@ -93,12 +97,13 @@ module.exports = class Fair extends Backbone.Model
     sections = new Backbone.Collection
     sections.fetch _.extend options,
       cache: true
+      cacheTime: DEFAULT_CACHE_TIME
       url: "#{@url()}/sections"
 
   fetchPrimarySets: (options) ->
     orderedSets = new OrderedSets
     orderedSets
-      .fetchItemsByOwner('Fair', @get('id'), cache: options.cache)
+      .fetchItemsByOwner('Fair', @get('id'), cache: options.cache, cacheTime: options.cacheTime)
```

Comment:
Why are you using the cache time from the `options` parameter in this method instead of using `DEFAULT_CACHE_TIME` like the other methods do?

This may cause unexpected behavior if we modify the cache time in `routes.coffee` (where `fetchPrimarySets` is called) but not in this file.

Reply:
Never mind. After reviewing a similar line of code in `src/desktop/collections/ordered_sets.coffee` this code makes sense.